### ISSUE TITLE
Only supports vue 3?

### ISSUE THREAD

**Issue by rochagbr** [pure-vue/vue-jss#3]
First, congrats by this lib :)
Have a question. This lib supports only vue 3?
Thankyou

**Comment by Jokcy** [comment]
Yeah, and by now I'm not sure if I will keep developing this lib. I'm more curious about emotion now.
